### PR TITLE
Implemented createIndex and dropIndex

### PR DESCRIPTION
**Pull request by skathuria**
Please review the code and let me know of changes.

### CODE REVIEW

File: src/main/java/io/polyglotted/elastic/common/EsRestClient.java [hunk 1]
```diff
@@ -74,15 +72,25 @@
         } catch (Exception e) { throw handleEx("getSettings failed", e); }
     }
 
-    @Override public void createIndex(CreateIndexRequest request) { throw new UnsupportedOperationException(); }
+    @Override public String createIndex(CreateIndexRequest request) {
+        try {
+            return performCliRequest("PUT","/"+request.index());
```
Comment (vshank77) [inline src/main/java/io/polyglotted/elastic/common/EsRestClient.java]: Please support settings, aliases and mappings for create index.. This is the core.

File: src/test/java/io/polyglotted/elastic/common/test/ElasticClientIntegTest.java [hunk 2]
```diff
@@ -1,24 +1,32 @@
...
 
 public class ElasticClientIntegTest {
 
     @Test
     public void testHighLevelClient() throws Exception {
+        String response = null;
         try (ElasticClient elasticClient = highLevelClient(esSettingsBuilder().userName("elastic").password("SteelEye").build())) {
             checkClusterHealth(elasticClient);
+            assertThat(elasticClient.createIndex(new CreateIndexRequest("customer")), containsString("acknowledged"));
+            assertThat(elasticClient.dropIndex("customer"), containsString("acknowledged"));
```
Comment (vshank77) [inline src/test/java/io/polyglotted/elastic/common/test/ElasticClientIntegTest.java]: happy to accept this now! But as we start implementing more and more features, this method needs refactoring heavily to read like a prose..

File: src/test/java/io/polyglotted/elastic/common/test/ElasticClientIntegTest.java [hunk 3]
```diff
@@ -1,24 +1,32 @@
...
         }
     }
 
+
```
Comment (vshank77) [inline src/test/java/io/polyglotted/elastic/common/test/ElasticClientIntegTest.java]: why many empty lines (please do Menu=>Code=>Reformat Code (cmd+alt+L) to clean up.. This needs both of us to have the same formatting style, so please remind me to share it with you tomorrow.

File: src/main/java/io/polyglotted/elastic/common/ElasticSettings.java [hunk 4]
```diff
@@ -15,6 +15,7 @@
     final String password;
     final int retryTimeoutMillis;
 
+
```
Comment (vshank77) [inline src/main/java/io/polyglotted/elastic/common/ElasticSettings.java]: Please remember my comment earlier regarding clean code and updating only whats necessary - if you're not modifying a code, it should be reverted

**Comment by skathuria** [comment]
Please review the changes made.

**Comment by skathuria** [comment]
Made the required formatting changes, please review.

**Pull request merged**